core/Cargo.toml

```diff
@@ -11,7 +11,6 @@ deps = { version = "0.1.0", path = "../deps" }
 
 anyhow = { version = "1.0" }
 bech32 = { version = "0.9" }
-cardano-multiplatform-lib = { git = "https://github.com/dcSpark/cardano-multiplatform-lib.git", rev = "7e930347d665b05875b7e2974be2f49e8cbe2a12" }
```

Comment:
there seem to be a couple of unused deps here (bech32, cryptoxide, hex, rand), that seem to be used in the moved module, but not sure if those were added because of the moved code and it's the right place to remove them.